docker/webpack-watch

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+npm install
+cd frontend
+
+exec $(npm bin)/webpack --colors --watch --progress
```

Comment:
I'm likely to run

```
% docker-compose run web bash
root@51585c3a30c4:/app# cd frontend && $(npm bin)/webpack --colors --watch --progress
```

I think: if it on my list of task that I want to do I need to know it explicitly (new users can be confuse about somethings, for example).

But it is on you :P